pkg/vm/engine/tae/containers/utils.go

```diff
@@ -74,18 +47,59 @@ func CloneWithBuffer(src Vector, buffer *bytes.Buffer, allocator ...*mpool.MPool
 	return
 }
 
+func FillBufferWithBytes(bs *Bytes, buffer *bytes.Buffer) *Bytes {
+	buffer.Reset()
+	size := bs.Size()
+	if buffer.Cap() < size {
+		buffer.Grow(size)
+	}
+	nbs := stl.NewBytesWithTypeSize(bs.TypeSize)
+	buf := buffer.Bytes()[:size]
+	copy(buf, bs.StorageBuf())
+	copy(buf[bs.StorageSize():], bs.HeaderBuf())
+
+	nbs.SetStorageBuf(buf[:bs.StorageSize()])
+	nbs.SetHeaderBuf(buf[bs.StorageSize():bs.Size()])
+	return nbs
+}
+
 func UnmarshalToMoVec(vec Vector) *movec.Vector {
 	bs := vec.Bytes()
 
 	mov, _ := movec.FromDNVector(vec.GetType(), bs.Header, bs.Storage)
 	if vec.HasNull() {
```

Comment:
Previously, the null structures of cn and dn were different, so new nulls needs to be constructed, and now they are the same, so I don't think it's necessary to do this now